src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/servicebus/ServiceBusHelper.java

```diff
@@ -27,7 +29,9 @@
 
     private final IQueueClient sendClient;
 
-    public ServiceBusHelper(QueueClientSupplier queueClientSupplier) {
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Could we inject the object mapper so that we get consistent jackson configuration?
I think that just instantiating it means the we get the default object mapper from jackson not the configured one from spring?

Reply:
metafile validator uses a local object mapper as it needs custom configuration but otherwise like @timja pointed out we can use default one provided by spring.